### PR TITLE
Add LZ4 compression/decompression

### PR DESCRIPTION
**Pull request by rharkes**
I've tried to add LZ4 compression in a similar way as was done before for Zstd decompression. However, LZ4 does not tell how large the decompressed data will be. So this is a required parameter in CodecOptions. But I'm not sure how to best enforce it. 

The reason I'd like to have LZ4 decompression is that it is used by the imaris fileformat, but it cannot be opened by bioformats at the moment.

### CODE REVIEW

File: src/main/java/ome/codecs/LZ4Codec.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        int len = compressor.compress(data,0, data.length, output, 0, output.length);
+        return Arrays.copyOfRange(output, 0, len);
+    }
+
+    /* @see BaseCodec#decompress(RandomAccessInputStream, CodecOptions) */
+    @Override
+    public byte[] decompress(RandomAccessInputStream in, CodecOptions options)
+            throws CodecException, IOException
+    {
+        long byteCount = in.length() - in.getFilePointer();
```
Comment (melissalinkert) [inline src/main/java/ome/codecs/LZ4Codec.java:69]: The `CodecOptions.maxBytes` option should be checked here. https://github.com/ome/ome-codecs/blob/c95aee4a3fc80910e0d209c6f2a404135d60b995/src/main/java/ome/codecs/JPEG2000Codec.java#L260 is an example of how to use that option.

File: src/test/java/ome/codecs/LZ4CodecTest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package ome.codecs;
```
Comment (melissalinkert) [inline src/test/java/ome/codecs/LZ4CodecTest.java:1]: The copyright statement used in other files in this repository should be added to the top of this file.

**Review by melissalinkert — CHANGES_REQUESTED** [review]
Thanks for this, @rharkes.

Before we more carefully consider this for inclusion, could you please submit a contributor license agreement as described in https://ome-contributing.readthedocs.io/en/latest/cla.html?

File: src/main/java/ome/codecs/LZ4Codec.java (hunk 1 above)
Comment (rharkes) [reply src/main/java/ome/codecs/LZ4Codec.java:69]: @melissalinkert could you maybe clarify?

Comment (melissalinkert) [reply src/main/java/ome/codecs/LZ4Codec.java:69]: Apologies, @rharkes, I think I misread this initially. I see now that `options.maxBytes` gets used in line 91, which makes sense.

**Comment by melissalinkert** [comment]
Added `include` label so we can verify that this doesn't impact tests (Friday or later, as I was too slow for Thursday's build).

**Review by melissalinkert — APPROVED** [review]
Thanks again, @rharkes. Merging this in preparation for a release of ome-codecs.

**Pull request merged**